testing/coreruleset_test.go

```diff
@@ -23,22 +21,22 @@ import (
 	"github.com/corazawaf/coraza/v3/seclang"
 )
 
-var crspath = ""
+var crspath string
 
 func init() {
 	fmt.Println("Preparing CRS...")
 	crs, err := downloadCRS("32e6d80419d386a330ddaf5e60047a4a1c38a160")
 	if err != nil {
-		panic(err)
+		panic(fmt.Sprintf("failed to download CRS: %s", err.Error()))
 	}
-	crspath, err = os.MkdirTemp(os.TempDir(), "crs")
+	tmpPath, err := os.MkdirTemp(os.TempDir(), "crs")
```

Comment:
crspath is a global variable that later changes in https://github.com/corazawaf/coraza/pull/414/files#diff-1f537de36e7969226678e652b7cc4fd1f88b8cedaec6cf2d5ffc365c656ebf46L174 so I decided to change this.